newton_usd_schemas/plugInfo.json

```diff
@@ -58,6 +58,20 @@
               "apiSchemaCanOnlyApplyTo": [
                 "Material"
               ]
+            },
+            "NewtonPhysicsKaminoSceneAPI": {
```

Comment:
Lets shuffle this up above CollisionAPI & just below XpbdAPI (after #18 is merged)